### PR TITLE
Add AdditionalPrinterColumns to cluster-api crds

### PR DESCRIPTION
**Pull request by alvaroaleman**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
More details are shown when using `kubectl get machine/machineset/machinedeployment`
```

### CODE REVIEW

File: api/pkg/controller/usercluster/resources/machine-controller/crds.go [hunk 1]
```diff
@@ -73,6 +167,44 @@ func MachineDeploymentCRDCreator() reconciling.NamedCustomResourceDefinitionCrea
...
+					Name:     "Deleted",
+					Type:     "date",
+					JSONPath: ".metadata.deletionTimestamp",
+					Priority: 0,
+				},
+				{
+					Name:     "Replicas",
+					Type:     "string",
+					JSONPath: ".spec.replicas",
+					Priority: 0,
```
Comment (mrIncompetent) [inline api/pkg/controller/usercluster/resources/machine-controller/crds.go]: How about adding `dep.status.availableReplicas` ?
That way people can see what the progress is during an update

File: api/pkg/controller/usercluster/resources/machine-controller/crds.go [hunk 2]
```diff
@@ -49,6 +99,50 @@ func MachineSetCRDCreator() reconciling.NamedCustomResourceDefinitionCreatorGett
...
```
Comment (mrIncompetent) [inline api/pkg/controller/usercluster/resources/machine-controller/crds.go]: https://github.com/kubermatic/kubermatic/pull/3364/files#r279286382

**Comment by alvaroaleman** [comment]
@mrIncompetent Updated, PTAL

**Pull request merged**